tests/unit/segments/test_segments_models.py

```diff
@@ -57,6 +57,7 @@
         (constants.NOT_CONTAINS, "bar", "baz", True),
         (constants.REGEX, "foo", r"[a-z]+", True),
         (constants.REGEX, "FOO", r"[a-z]+", False),
+        (constants.REGEX, "1.2.3", r"\d", True),
```

Comment:
This should probably have been in another PR?

Reply:
No, it actually makes sure that we don't treat anything that looks like a SemVer as SemVer